opentasks/src/main/java/org/dmfs/tasks/widget/TextFieldView.java

```diff
@@ -106,4 +106,9 @@ public void onContentChanged(ContentSet contentSet)
 			}
 		}
 	}
+
+	public String getText()
+	{
+		return mText.getText().toString();
```

Comment:
Don't use UI components for business logic. UI is either output or input, you're using it like a variable.